Makefile

```diff
@@ -201,12 +204,33 @@ xgo:
 install-os-dependencies:
 	_assets/scripts/install_deps.sh
 
-setup: install-os-dependencies dep-install lint-install mock-install gen-install update-fleet-config ##@other Prepare project for first build
+setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config ##@other Prepare project for first build
 
 generate: ##@other Regenerate assets and other auto-generated stuff
 	go generate ./static ./static/migrations
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
+prepare-release: clean-release
+	mkdir -p $(RELEASE_DIRECTORY)
+	mv build/bin/statusgo-android-16.aar $(RELEASE_DIRECTORY)/status-go-android.aar
+	#mv build/bin/statusgo-ios-9.3-framework/status-go-ios.zip $(RELEASE_DIRECTORY)/status-go-ios.zip
+	${MAKE} clean
+	zip -r $(RELEASE_DIRECTORY)/status-go-desktop.zip . -x *.git*
+
+clean-release:
+	rm -rf $(RELEASE_DIRECTORY)
+
+release:
+	@read -p "Are you sure you want to create a new GitHub release? (y/n): " REPLY; \
+	if [ $$REPLY = "y" ]; then \
+	    github-release "status-im/status-go" "$(RELEASE_TAG)" "$(release_branch)" "" "$(RELEASE_DIRECTORY)/*"; \
```

Comment:
After giving it a second though, maybe we can provide a default value for this?

```
release: RELEASE_BRANCH = develop
release: PRE_RELEASE = 0
release:
    @read -p "Are you sure you want to create a new GitHub $(shell if [[ "$$PRE_RELEASE" == "0" ]] ; then echo release; else echo pre-release ; fi) against $(RELEASE_BRANCH) branch? (y/n): " REPLY; \
    ...
```

Reply:
What's gonna happen if `RELEASE_TAG` already exists?

Reply:
it will continue uploading the files, but it will stop if the files are already there, which seems like sensible behavior (we don't allow to override the assets, but in case they haven't uploaded at all, we allow to resume)

Reply:
That sounds good!